docs/developers/maintenance.md

```diff
@@ -0,0 +1,61 @@
+# Maintenance
+
+This document describes the maintenance tasks that need to be performed from time to time. 
+To rare to depend only on personal memory. 
+
+## Refreshing tokens for the auto upgrade of test constraints and vendored packages. 
+
+Because of the security GitHub policy, the commits and pull requests created by an action that uses default `GITHUB_TOKEN`
+will not trigger another action's runs. The possible workaround for this is to close and then reopen the pull request. 
+But this requires additional actions by core devs.
+
+
+To get automatically running workflows, we need to create a personal access token (PAT) and add it to the repository secrets.
+For security reasons, we decide to create a fine-grained token. It allows us to provide only the required permissions.
+
+The token has a one-year expiration date. After that, it needs to be refreshed.
+
+The token is stored in `GHA_TOKEN` and the required permissions are:
+
+ * **read** to repository metadata
+ * **read and write** to pull requests
+ * **read and write** to code
```

Comment:
If I understand correctly we are letting people know this is how they should do it but these are not default options, so I would phrase as
```suggestion
The token should have a one-year expiration date. After that, it needs to be refreshed.

The token should be named `GHA_TOKEN` and the required permissions are:

 * **read** to repository metadata
 * **read and write** to pull requests
 * **read and write** to code
```